frontend/src/components/queue/JoinQueue.jsx

```diff
@@ -105,16 +108,58 @@ function JoinQueue(props) {
       request.subject = subject;
       request.question = details;
       request.location = location;
+      request.code = code;
       request.groupOption = groupOption.toString();
 
-
       //alert(groupoption);
-      joinQueue(request, url); 
+      /* Makes post and checks success */
+      const joinResponse = await joinQueue(request, url);
+
+      if (joinResponse.success && !groupOption) {
```

Comment:
This logic seems to have changed. What happens if they want to be in a group?  Do they still get put on the queue?